### PR TITLE
Fix event and remove socket from exclude plugin default

### PR DESCRIPTION
**Pull request by niicojs**
### Short Description:

Fix event priority that broke pokemon_caught event (#270).  
Remove socket from exclude plugin default

@OpenPoGo/maintainers

### CODE REVIEW

File: config/config.yml.example [hunk 1]
```diff
@@ -59,7 +59,7 @@ movement:
 plugins:
     # Do not automatically load the specified plugins
     exclude:
-        - socket
+        - pluginname
```
Comment (andythorne) [inline config/config.yml.example]: I think a lot of people are just going to rename this file, so i would leave exclude an an empty array. maybe update the comments to show it should be the plugin name.

File: plugins/socket/botevents.py [hunk 2]
```diff
@@ -80,16 +78,19 @@ def pokestop_visited_event(self, bot=None, pokestop=None):
         }
         self.socketio.emit("pokestop_visited", emitted_object, namespace="/event")
 
-    def pokemon_caught(self, bot=None, pokemon=None, position=None):
+    def pokemon_caught_event(self, bot=None, pokemon=None, position=None):
+        print "pokemon_caught"
```
Comment (andythorne) [inline plugins/socket/botevents.py]: Remove prints (line 84 and 90 as well)

File: config/config.yml.example (hunk 1 above)
Comment (niicojs) [inline config/config.yml.example]: The thing is the syntax for nothing ([]) is so much different that I think we need a real example with a fake name. Discussion on slack seems to prove that.

File: plugins/socket/botevents.py (hunk 2 above)
Comment (niicojs) [inline plugins/socket/botevents.py]: Oh yes sorry

File: config/config.yml.example (hunk 1 above)
Comment (andythorne) [inline config/config.yml.example]: imo if people can't figure out `[]` is an empty array then ¯_(ツ)_/¯

`['pluginname']` is also a valid array, using `-`s are just a multiline array.

**Comment by andythorne** [comment]
:+1: 

[Approved with PullApprove](https://pullapprove.com/OpenPoGo/OpenPoGoBot/pull-request/325/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=OpenPoGo/OpenPoGoBot)

**Comment by MaxLeiter** [comment]
:+1:

[Approved with PullApprove](https://pullapprove.com/OpenPoGo/OpenPoGoBot/pull-request/325/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=OpenPoGo/OpenPoGoBot)

**Pull request merged**